bootstrap.py

```diff
@@ -174,12 +167,18 @@ def exec_failexit(command):
     print_success(command)
     print ""
 
+def yum(command, pkgs = ""):
+    for x in range(0, int(options.download)):
+        if exec_failok("/usr/bin/yum -y --downloadonly %s %s" % (command, pkgs)):
```

Comment:
Yeah, I know. But since this is optional flag which is turned of by default, can we assume user knows this (otherwise yum will print an error which will be ignored anyway).